modules/react/card/lib/Card.tsx

```diff
@@ -60,28 +58,21 @@ const Box = styled('div', {
   ({height}) => height && {height}
 );
 
-const Header = styled('h3')(type.h3, {
-  marginBottom: space.m,
-  marginTop: 0,
-});
-
-const Body = styled('div')(type.body);
-
-export default class Card extends React.Component<CardProps> {
-  public render() {
-    const {
-      depth = depthValues[2],
-      padding = space.l,
-      heading,
-      headingId,
-      ...elemProps
-    } = this.props;
-
+export const Card = createComponent('div')({
+  displayName: 'Card',
+  Component: (
+    {depth = depthValues[2], children, padding = space.l, ...elemProps}: CardProps,
+    ref,
+    Element
+  ) => {
     return (
-      <Box {...elemProps} depth={depth} padding={padding}>
-        {heading && <Header id={headingId}>{heading}</Header>}
-        <Body>{this.props.children}</Body>
+      <Box ref={ref} as={Element} depth={depth} padding={padding} {...elemProps}>
```

Comment:
Nit: is it worth renaming this due to the upcoming release of `Box` from @alanbsmith?

Reply:
I figured this would be refactored to use Box when it comes out.

Reply:
Yeah, agree. Eventually Card will be built with Box. This is fine for now.